ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -275,6 +275,125 @@ objects:
+            date >> /tmp/artifacts/bootstrap/bootkube.log
+            sleep 5
+          done
+        }
+
+        function stream-kubectl-get-all () {
+          ip=${1}
+          while true; do
+            ssh -o "StrictHostKeyChecking=no" core@${ip} sudo kubectl get all --kubeconfig=/var/opt/tectonic/auth/kubeconfig -a --all-namespaces -w 2>&1 >> /tmp/artifacts/bootstrap/kubectl-get-all.log
+            date >> /tmp/artifacts/bootstrap/kubectl-get-all.log
+            sleep 5
+          done
+        }
+
+        function stream () {
+          # wait for terraform to start up
+          while [ ! -f /tmp/artifacts/installer/terraform.tfstate ]; do sleep 5; done
+
+          # wait for the bootstrap node to show up with an IP
+          ip=""
+          while [ -z "${ip}" ]; do
+            ip=$(terraform state show -state=terraform.tfstate module.bootstrap.aws_instance.bootstrap | sed -n 's/^public_ip *= *//p')
+          done
+
+          # try to login
+          while ! ssh -o "StrictHostKeyChecking=no" core@${ip} /bin/true; do
+            sleep 5
+          done
+
+          # start streaming
+          mkdir -p /tmp/artifacts/bootstrap
+          stream-bootkube ${ip} &
+          stream-kubectl-get-all ${ip} &
+        }
+
+        stream &
+        for i in `seq 1 120`; do
+          if [[ -f /tmp/shared/exit ]]; then
+            exit 0
+          fi
+          sleep 60 & wait
+        done
+
+    # Stream "kubectl get all -a --all-namespaces" into artifact
+    - name: stream-kubectl-get-all
+      image: ${IMAGE_INSTALLER}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /etc/openshift-installer
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        mkdir -p .ssh
+        chmod 700 .ssh
+        cp /etc/openshift-installer/ssh-privatekey .ssh/id_rsa
+        cp /etc/openshift-installer/ssh-publickey .ssh/id_rsa.pub
+
+        function stream () {
+          # wait for terraform to start up
+          while [ ! -f /tmp/artifacts/installer/terraform.tfstate ]; do sleep 5; done
+
+          # wait for the bootstrap node to show up with an IP
+          ip=""
+          while [ -z "${ip}" ]; do
+            ip=$(terraform state show -state=terraform.tfstate module.bootstrap.aws_instance.bootstrap | sed -n 's/^public_ip *= *//p')
+          done
+
+          # try to login
+          while ! ssh -o "StrictHostKeyChecking=no" core@${ip} /bin/true; do
+            sleep 5
+          done
+
+          # stream bootkube into artifact file
+          mkdir -p /tmp/artifacts/bootstrap
+          ssh -o "StrictHostKeyChecking=no" core@${ip} sudo journal -u bootkube 2>&1 > /tmp/artifacts/bootstrap/bootkube.log
```

Comment:
This container is named `stream-kubectl-get-all`, but it is streaming `journal -u bootkube`?  And it looks like your earlier container isstreaming bootkube too?

Reply:
typo. Had planned two containers, but then noticed the amount of shared code and merged. will fix.

Reply:
fixed